app/adapters/bookmarklet.js

```diff
@@ -16,12 +16,10 @@ export default BasicAdapter.extend({
     this.get('inspectedWindow').postMessage(options, this.get('inspectedWindowURL'));
   },
 
-  _connect: function() {
-    let self = this;
-
+  _connect: () => {
```

Comment:
ops, I actually wanted to restore it to it's original state, Any functions with a trailing `.`  (eg `.on()`, `.property()` etc) gives me an error (unexpected `.`) when trying to refactor them!
Any workaround/suggestion for this?